Lombiq.ChartJs.Tests.UI/Lombiq.ChartJs.Tests.UI.csproj

```diff
@@ -29,8 +29,12 @@
     <ProjectReference Include="..\..\..\..\test\Lombiq.UITestingToolbox\Lombiq.Tests.UI\Lombiq.Tests.UI.csproj" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(NuGetBuild)' == 'true'">
+    <DebugType>embedded</DebugType>
```

Comment:
Document here and in the workflow why we're doing this. (If we can get aware without having this change in other projects than this UI test one, see my other comment, then the workflow can be restored; it's no issue if for just this project we'll publish an empty symbols package.)

Reply:
(Just to make sure: the docs here are still needed, despite our other convo.)

Reply:
It's clear :D I'm almost done with it.